tests/ha/test_multi_server_pod_failure.py

```diff
@@ -230,8 +234,10 @@ def test_reads_kserver_pods_fail(self):
         LOGGER.info("Step 2: Performed READs and verified DI on the written data")
 
         LOGGER.info("Step 3: Shutdown %s server pod by deleting deployment (unsafe)", self.kvalue)
-        LOGGER.info("Get pod names to be deleted")
+        LOGGER.info("Get server pod names to be deleted")
         pod_list = self.node_master_list[0].get_all_pods(pod_prefix=const.SERVER_POD_NAME_PREFIX)
+        server_data_pod_list = pod_list.append(self.node_master_list[0].get_all_pods
+                                               (pod_prefix=const.POD_NAME_PREFIX))
```

Comment:
This is only data pod list right?

Reply:
i am appending the "data pod list" to server pod list